volumina/sliceSelectorHud.py

```diff
@@ -517,12 +536,15 @@ def createQuadViewStatusBar(self,
         self.timeSlider = QSlider(Qt.Horizontal)
         self.timeSlider.setMinimumWidth(10)
         self.timeSlider.setMaximumWidth(200)
+        self.timeSlider.setToolTip("Choose the time coordinate of the current view.")
```

Comment:
Why not call the function you just defined (setToolTipTimeSliderCrop) for this? It will be easier to change the wording in future if it's all in one place.